src/Akeneo/Pim/Enrichment/Component/Product/Security/ProductScopeMapper.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Pim\Enrichment\Component\Product\Security;
+
+use Akeneo\Connectivity\Connection\Infrastructure\Apps\Security\ScopeMapperInterface;
+
+/**
+ * @copyright 2021 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class ProductScopeMapper implements ScopeMapperInterface
```

Comment:
```suggestion
class ProductsScopeMapper implements ScopeMapperInterface
```
same here, with or without `s` because it's not the same for acls and scopes